app/models/setting/auth.rb

```diff
@@ -26,6 +26,8 @@ def self.default_settings
       self.set('idle_timeout', N_("Log out idle users after a certain number of minutes"), 60, N_('Idle timeout')),
       self.set('bcrypt_cost', N_("Cost value of bcrypt password hash function for internal auth-sources (4-30). Higher value is safer but verification is slower particularly for stateless API calls and UI logins. Password change needed to take effect."), 4, N_('BCrypt password cost')),
       self.set('bmc_credentials_accessible', N_("Permits access to BMC interface passwords through ENC YAML output and in templates"), true, N_('BMC credentials access')),
+      self.set('keycloak_jwks_url', N_("Keycloak JWKs URL, this url can be found at the /.well-known/openid-configuration path of your server. For example if you are using Keycloak this url would be found as `jwk_url` at https://<your keycloak server>/auth/realm/<realm name>/.well-known/openid-configuration"), nil, N_('Keycloak JWKs URL')),
```

Comment:
This should work with every implementation of OpenIDConnect, right? Wouldn't it be better to call this `oidc_jwks_url` so it's generic and not only for keycloak?

Reply:
@timogoebel you were correct, every IDP has an endpoint for the JWKS URL that returns a JSON. I hope this [comment](https://github.com/theforeman/foreman/pull/6549/files#diff-6afdff346c7e417ea7f74f437487706aR29) makes it crystal clear.